src/components/inputs/Upload.tsx

```diff
@@ -167,9 +170,20 @@ export const GenericImageUpload = <FormValues extends FieldValues>({
         onConfirm={() => dialogConfirmCrop()}
         open={dialogOpen}
         titleText='Tilpass bildet'>
-        <CropperWrapper>
-          <Cropper aspect={ratio} crop={crop} image={imageSrc} onCropChange={setCrop} onCropComplete={onCropComplete} onZoomChange={setZoom} zoom={zoom} />
-        </CropperWrapper>
+        <>
+          <CropperWrapper>
+            <Cropper
+              aspect={ratioFloat}
+              crop={crop}
+              image={imageSrc}
+              onCropChange={setCrop}
+              onCropComplete={onCropComplete}
+              onZoomChange={setZoom}
+              zoom={zoom}
+            />
+          </CropperWrapper>
+          <Typography textAlign='center'>Bildet bør være på formatet: {ratio}</Typography>
```

Comment:
Hva tror du om denne teksten istedet?
```suggestion
          <Typography textAlign='center'>Anbefalt størrelsesforhold: {ratio}</Typography>
```